### ISSUE TITLE
Test failures in master (Chrome 45)

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium#3004]
In Chrome 45 on Mac.  I did not test other platforms.

```
Scene/ScreenSpaceCameraController zoom in 2D
Expected 0 to be less than 0.
Error: Expected 0 to be less than 0.
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toBeLessThan (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/ScreenSpaceCameraControllerSpec.js:304:28
    at Object.<anonymous> (http://localhost:8080/Specs/spec-main.js:137:30)
    at attemptAsync (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1793:24)
    at QueueRunner.run (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1746:26)
Scene/ScreenSpaceCameraController zoom with max zoom rate in 2D
Expected 0 to be less than 0.
Error: Expected 0 to be less than 0.
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toBeLessThan (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/ScreenSpaceCameraControllerSpec.js:367:28
    at Object.<anonymous> (http://localhost:8080/Specs/spec-main.js:137:30)
    at attemptAsync (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1793:24)
    at QueueRunner.run (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1746:26)
Scene/TextureAtlas renders a four image atlas
Expected [ 0, 0, 0, 255 ] to equal [ 0, 255, 0, 255 ].
Error: Expected [ 0, 0, 0, 255 ] to equal [ 0, 255, 0, 255 ].
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toEqual (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/TextureAtlasSpec.js:348:39
    at http://localhost:8080/Source/ThirdParty/when.js:196:34
    at Object.then (http://localhost:8080/Source/ThirdParty/when.js:352:11)
    at Function.all (http://localhost:8080/Source/ThirdParty/when.js:507:42)
Expected [ 0, 0, 0, 255 ] to equal [ 0, 0, 255, 255 ].
Error: Expected [ 0, 0, 0, 255 ] to equal [ 0, 0, 255, 255 ].
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toEqual (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/TextureAtlasSpec.js:349:39
    at http://localhost:8080/Source/ThirdParty/when.js:196:34
    at Object.then (http://localhost:8080/Source/ThirdParty/when.js:352:11)
    at Function.all (http://localhost:8080/Source/ThirdParty/when.js:507:42)
Scene/TextureAtlas renders a two image atlas with non-zero borderWidthInPixels that resizes
Expected [ 0, 0, 0, 255 ] to equal [ 0, 255, 0, 255 ].
Error: Expected [ 0, 0, 0, 255 ] to equal [ 0, 255, 0, 255 ].
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toEqual (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/TextureAtlasSpec.js:620:48
    at http://localhost:8080/Source/ThirdParty/when.js:196:34
    at Object.then (http://localhost:8080/Source/ThirdParty/when.js:352:11)
    at Function.all (http://localhost:8080/Source/ThirdParty/when.js:507:42)
```

**Comment by mramato** [comment]
These are on Windows as well and I believe are regressions with Chrome 45 (which also introduced some other bugs as well).

**Comment by pjcozzi** [comment]
After the fixes in #3011, Chrome 45 is down to two test failures:
```
Scene/ScreenSpaceCameraController zoom in 2D
Expected 0 to be less than 0.
Error: Expected 0 to be less than 0.
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toBeLessThan (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/ScreenSpaceCameraControllerSpec.js:304:28
    at Object.<anonymous> (http://localhost:8080/Specs/spec-main.js:137:30)
    at attemptAsync (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1793:24)
    at QueueRunner.run (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1746:26)
Scene/ScreenSpaceCameraController zoom with max zoom rate in 2D
Expected 0 to be less than 0.
Error: Expected 0 to be less than 0.
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.Env.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toBeLessThan (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/ScreenSpaceCameraControllerSpec.js:367:28
    at Object.<anonymous> (http://localhost:8080/Specs/spec-main.js:137:30)
    at attemptAsync (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1793:24)
    at QueueRunner.run (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1746:26)
```

**Comment by pjcozzi** [comment]
I no longer see these on Mac or Windows in Chrome 46.
